.github/workflows/angular.yml

```diff
@@ -39,9 +39,9 @@ env:
   JHI_RUN_APP: 1
   JHI_JDK: 11
   JHI_LIB_REPO: https://github.com/jhipster/jhipster-bom.git
-  JHI_LIB_BRANCH: release
+  JHI_LIB_BRANCH: spring-boot-2.4
```

Comment:
I guess that it should be modified if we merge it into main

Reply:
Yes, these branches are just temporary until all tests are passing.